lib/run.rb

```diff
@@ -1,18 +1,28 @@
 require_relative './input'
 require_relative './prompt'
+require_relative './board_presenter'
 
 def prompt_players
     puts Prompt::WELCOME
 end
 
 def get_players
-    input = Input.new.get_input
+    Input.new.get_input
 end
 
 def prompt_token_selection
     puts Prompt::MAKE_TOKEN_SELECTION
 end
 
 def get_player_token
-    input = Input.new.get_input
+    Input.new.get_input
 end
+
+def display_board
+    BoardPresenter.new.display_board
+end
+
+def assign_token
+    get_player_token == "X" ? "O": "X"
+end
+
```

Comment:
So, when you run `ruby run.rb` .. What currently happens? I don't see anything in here where these methods are actually called. So the acceptance criteria isn't actually met. Am I missing something?